proxy/http/Http1ClientTransaction.cc

```diff
@@ -67,6 +67,7 @@ Http1ClientTransaction::transaction_done()
   // If the parent session is not in the closed state, the destroy will not occur.
   if (parent) {
     parent->destroy();
+    parent = NULL;
```

Comment:
Yeah, I'm not 100% sure this is right or not. But from the backtrace, `destroy()` calls `free()`. In this case we need to set parent NULL.

```
#0  Http1ClientSession::free (this=0x60ae0001cc80) at Http1ClientSession.cc:100
#1  0x00000000005c2386 in ProxyClientSession::handle_api_return (this=0x60ae0001cc80, event=60000) at ProxyClientSession.cc:206
#2  0x00000000005c1f25 in ProxyClientSession::do_api_callout (this=0x60ae0001cc80, id=TS_HTTP_SSN_CLOSE_HOOK) at ProxyClientSession.cc:177
#3  0x000000000065f787 in Http1ClientSession::destroy (this=0x60ae0001cc80) at Http1ClientSession.cc:93
#4  0x0000000000664d20 in Http1ClientTransaction::transaction_done (this=0x60ae0001cf60) at Http1ClientTransaction.cc:69
```